engine/language_client_codegen/src/ruby/generate_types.rs

```diff
@@ -30,7 +29,7 @@ pub(crate) struct RubyStreamTypes<'ir> {
     partial_classes: Vec<PartialRubyStruct<'ir>>,
 }
 
-/// The Python class corresponding to Partial<TypeDefinedInBaml>
+/// The Python class corresponding to Partial<TypeDefinedjInBaml>
```

Comment:
typo!

Reply:
@hellovai in the next PR, I will correct it, my bad